invoice/src/main/java/org/killbill/billing/invoice/InvoiceDispatcher.java

```diff
@@ -700,4 +705,41 @@ public BillingActionPolicy getBillingActionPolicy() {
             return null;
         }
     }
+
+    public void processParentInvoiceForInvoiceGeneration(final ImmutableAccountData account, final UUID invoiceId, final InternalCallContext context) throws InvoiceApiException {
```

Comment:
Could you rename the variables to make it obvious what refers to the parent, and what refers to the child?